marketplace/charts/marketplace-hooks/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
The reason why I don't want to call this chart "hooks" is because we might extend it to include more in the future, for example a controller. The idea here is this chart supports the application to run in GCP Marketplace.

How about one of the following:
- marketplace-app-support
- marketplace-application
- application-support

Reply:
Sync'd offline - moving forward with `marketplace-integration`.